src/components/input/input.js

```diff
@@ -59,8 +58,8 @@ Input.propTypes = {
 };
 
 Input.defaultProps = {
-  value: '',
-  className: null,
+  value: '', // @todo: why undefined creates an uncontrolled/controlled component switch?
```

Comment:
So from what I understand if you put "undefined" React doesn't even put the value attribute ON the HTML input field which forfeits React from controlling the 1-way input field.  This makes sense because this is a Text field and "undefined" isn't really a valid "string" per se.  So empty string actually is correct.

Reply:
Thanks, makes sense!